### PR TITLE
[pat] Change title from Personal Access Tokens to Access Tokens

### PR DESCRIPTION
**Pull request by easyCZ**
## Description

From
Screenshot 2022-12-02 at 10 42 38

To
Screenshot 2022-12-02 at 10 42 49

## Related Issue(s)

Fixes #

## Release Notes

```release-note
NONE
```

## Werft options:

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [ ] /werft with-preview
- [ ] /werft with-large-vm
- [ ] /werft with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`

### CODE REVIEW

File: components/dashboard/src/settings/PersonalAccessTokens.tsx [hunk 1]
```diff
@@ -50,7 +50,7 @@ const personalAccessTokenNameRegex = /^[a-zA-Z0-9-_ ]{3,63}$/;
 
 enum TokenAction {
     Create = "CREATED",
-    Regerenrate = "REGENERATED",
+    Regenerate = "REGENERATED",
```
Comment (easyCZ) [inline components/dashboard/src/settings/PersonalAccessTokens.tsx:53]: drive-by typo fix

File: components/dashboard/src/settings/PersonalAccessTokens.tsx [hunk 2]
```diff
@@ -534,7 +534,7 @@ function ListAccessTokensView() {
         <>
             <div className="flex items-center sm:justify-between mb-4">
                 <div>
-                    <h3>Personal Access Tokens</h3>
+                    <h3>Access Tokens</h3>
```
Comment (gtsiolis) [inline components/dashboard/src/settings/PersonalAccessTokens.tsx:537]: **nitpick**: Since you're changing this, could we also update the empty state and drop the `PAT` acronym?

https://github.com/gitpod-io/gitpod/blob/d4f97802102e0c45e61e25a0bd8af6f42e305167/components/dashboard/src/settings/PersonalAccessTokens.tsx#L605

**Review by gtsiolis — COMMENTED** [review]
Hi, @easyCZ! 

homer-simpson

File: components/dashboard/src/settings/PersonalAccessTokens.tsx (hunk 2 above)
Comment (easyCZ) [reply components/dashboard/src/settings/PersonalAccessTokens.tsx:537]: Done.

Comment (gtsiolis) [reply components/dashboard/src/settings/PersonalAccessTokens.tsx:537]: Thanks, @easyCZ! 🍊

Comment (gtsiolis) [inline components/dashboard/src/settings/PersonalAccessTokens.tsx:537]: **nitpick**: Few more. Last comment, promise.

https://github.com/gitpod-io/gitpod/blob/6386b4bcf484bfe87b8938580b8fe696c7ff0d46/components/dashboard/src/settings/PersonalAccessTokens.tsx#L340

https://github.com/gitpod-io/gitpod/blob/6386b4bcf484bfe87b8938580b8fe696c7ff0d46/components/dashboard/src/settings/PersonalAccessTokens.tsx#L413

https://github.com/gitpod-io/gitpod/blob/6386b4bcf484bfe87b8938580b8fe696c7ff0d46/components/dashboard/src/settings/PersonalAccessTokens.tsx#L654

https://github.com/gitpod-io/gitpod/blob/6386b4bcf484bfe87b8938580b8fe696c7ff0d46/components/dashboard/src/settings/PersonalAccessTokens.tsx#L666

Comment (easyCZ) [reply components/dashboard/src/settings/PersonalAccessTokens.tsx:537]: Interesting, looks like these must have resulted from a semantic merge as I removed them previously. Thanks for spotting, will get on these!

Comment (easyCZ) [reply components/dashboard/src/settings/PersonalAccessTokens.tsx:537]: updated

**Review by laushinka — APPROVED** [review]
